docs/imports/rock-the-vote.md

```diff
@@ -0,0 +1,120 @@
+
+So, the Alpha sends their page to a Beta and they register. The Alpha's referral links look like this: https://vote.dosomething.org/member-drive?userId={userId}&r=user:{userId},campaign:{campaignID},campaignRunID={campaignRunID},source=web,source_details=onlinedrivereferral,referral=true
+
+We've added `referral=true` to the link so that we can know to not attribute the registration to the NS ID that is present in the URL. In this case, this NS ID is the referrer and not the registrant.
+
+If the referral column has `referral=true` in it, then proceed with the logic with dealing w/ non-member registrants above.
+
+### How to count these as impact
+
+Based on the above statuses, some should be counted as a RB and some should not. This determination was made by the executive team and allows us to report internally progress towards the organization's report back goal. Here's what counts as a reportback from the RTV export:
+
+- `register-form`
+- `register-OVR`
+
+Note: `register-form` and `register-OVR` are the only statuses that count as _registrations_.
+
+Rogue will NOT store this information, but will return a derived value in the JSON response when the voter registration post is created or read that holds this information. The logic to determine this is as follows:
+
+```php
+if (in_array($rogueStatus, ['confirmed', 'register-form', 'register-OVR'])) {
+    $reportbackStatus = 'T';
+} else {
+    $reportbackStatus = 'F';
+}
+```
+
+## Notes
+
+- Data uses the post `details` to determine `source` and `source_detail` used in voter registration reporting.
+- The `submission_created_at` date is when the importer ran. Details about when the registration was created/updated are in the `source_details`.
+- All of these signups will have a `source` of `importer-client` (this is how messaging is suppressed in C.io)
+- In early iterations of the import, the month that the registration came in would inform the `action` column (e.g., february-2018-rockthevote)
+- In early iterations of the import, we would pass Campaign/Run IDs as parameters within the referral code to use when upsert a  `voter-reg` post.
+- If a user shares their UTM'ed URL with other people, there could be duplicate referral codes but associated with different registrants:
+  See a [screenshot](https://cl.ly/0v210N283y2X) of what this data looks like (note: the user depicted in this spreadsheet is fake.)
+
+## Open Questions
+
+1. How can we distinguish between TV and RTV import? Do we need to?
+2. RTV has more birthdays in it, can we use that on the NS profile when we create accounts? (TV didn't provide this)
```

Comment:
This sounds a bit like https://www.pivotaltracker.com/story/show/170075407.

Reply:
@aaronschachter I'm unable to follow that link, it says:
```
Sorry, we couldn't find that story
It may be in a project you haven't been invited to yet or it may have been deleted.
```

Reply:
@aaronschachter yup, that pivotal captures the RTV birthday work. We can distinguish between TV & RTF based on the `source_detail`s, I believe